src/app/routes/cpsAsset/getInitialData/addHeadlineBlock/index.js

```diff
@@ -22,7 +22,9 @@ const getHeadlineBlocks = json => {
 };
 
 const firstBlockIsVideo = blocks =>
-  ['video', 'version', 'legacyMedia'].includes(path(['0', 'type'], blocks));
+  ['video', 'version', 'legacyMedia', 'unavailableMedia'].includes(
```

Comment:
Can the type also be `media`? If so, should we include it in the list?

Reply:
Media will never be included in this list because it gets converted from `media` -> `video` when converting to optimo blocks.  No further action required.

Reply:
👍